src/main/java/apoc/export/graphml/XmlGraphMLReader.java

```diff
@@ -60,30 +62,82 @@ public XmlGraphMLReader reporter(Reporter reporter) {
             Object parse(String value) {
                 return Boolean.valueOf(value);
             }
+
+            Object parseList(String value) {
+                List parsedList = JsonUtil.parse(value, null, List.class);
```

Comment:
we could generify the functions
there is a method in Java lang class to create an array of the type.
and then only call.
`return parseList(value, Boolean.class) `